### PR TITLE
RAD-181: Update guide_star, instrument, and photometry schemas

### LINKED ISSUE #470: Update guide_star, instrument, and photometry schemas
_Issue [RAD-181](https://jira.stsci.edu/browse/RAD-181) was created on JIRA by [Paul Huwe](https://jira.stsci.edu/secure/ViewProfile.jspa?name=phuwe):_

Update the guide_star, instrument, and photometry schemas to match RTB documentation

### PR DESCRIPTION
**Pull request by PaulHuwe**
Resolves [RAD-181](https://jira.stsci.edu/browse/RAD-181)

Closes #470 

This PR update guide_star, instrument, and photometry schemas.

## Tasks
- [ ] Update or add relevant `rad` tests.
- [ ] Update relevant docstrings and / or `docs/` page.
- [x] Does this PR change any schema files?
  - [x] Schema changes were discussed at RAD Review Board meeting.
- [x] Does this PR change any API used downstream? (If not, label with `no-changelog-entry-needed`.)
  - [ ] Write news fragment(s) in `changes/`: `echo "changed something" > changes/<PR#>.<changetype>.rst` (see below for change types).
  - [ ] Start a `romancal` regression test (https://github.com/spacetelescope/RegressionTests/actions/workflows/romancal.yml) with this branch installed (`"git+https://github.com/<fork>/rad@<branch>"`).
  - [x] Update relevant `roman_datamodels` utilities and tests.

<details><summary>News fragment change types:</summary>

- ``changes/<PR#>.feature.rst``: new feature
- ``changes/<PR#>.bugfix.rst``: fixes an issue
- ``changes/<PR#>.doc.rst``: documentation change
- ``changes/<PR#>.removal.rst``: deprecation or removal of public API
- ``changes/<PR#>.misc.rst``: infrastructure or miscellaneous change
</details

### CODE REVIEW

File: src/rad/resources/schemas/guidestar-1.0.0.yaml [hunk 1]
```diff
@@ -159,129 +178,129 @@ properties:
...
     type: number
     sdf:
       special_processing: VALUE_REQUIRED
       source:
         origin: TBD
     archive_catalog:
       datatype: float
-      destination: [WFIExposure.gs_dec, GuideWindow.gs_dec]
-  gs_ura:
+      destination: [WFIExposure.dec, GuideWindow.dec]
```
Comment (kdupriestsci) [inline src/rad/resources/schemas/guidestar-1.0.0.yaml:207]: I believe there's already a 'dec' field in WFIExposure and GuideWindow from file target-1.0.0.yaml

File: src/rad/resources/schemas/guidestar-1.0.0.yaml [hunk 2]
```diff
@@ -159,129 +178,129 @@ properties:
...
     type: number
     sdf:
       special_processing: VALUE_REQUIRED
       source:
         origin: TBD
     archive_catalog:
       datatype: float
-      destination: [WFIExposure.gs_ra, GuideWindow.gs_ra]
-  gs_dec:
+      destination: [WFIExposure.ra, GuideWindow.ra]
```
Comment (kdupriestsci) [inline src/rad/resources/schemas/guidestar-1.0.0.yaml:194]: I believe there's already an "ra" field in WFIExposure and GuideWindow from file target-1.0.0.yaml

File: src/rad/resources/schemas/guidestar-1.0.0.yaml [hunk 3]
```diff
@@ -159,129 +178,129 @@ properties:
...
     type: number
     sdf:
       special_processing: VALUE_REQUIRED
       source:
         origin: TBD
     archive_catalog:
       datatype: float
-      destination: [WFIExposure.gs_ura, GuideWindow.gs_ura]
-  gs_udec:
+      destination: [WFIExposure.ra_uncertainty, GuideWindow.ra_uncertainty]
```
Comment (kdupriestsci) [inline src/rad/resources/schemas/guidestar-1.0.0.yaml:220]: I believe there's already a 'ra_uncertainty' field in WFIExposure and GuideWindow from file target-1.0.0.yaml

File: src/rad/resources/schemas/guidestar-1.0.0.yaml [hunk 4]
```diff
@@ -159,129 +178,129 @@ properties:
...
+      Uncertainty in the guide star declination from the
+      Guide Star Catalog (GSC) in units of degrees.
     type: number
     sdf:
       special_processing: VALUE_REQUIRED
       source:
         origin: TBD
     archive_catalog:
       datatype: float
-      destination: [WFIExposure.gs_udec, GuideWindow.gs_udec]
```
Comment (kdupriestsci) [inline src/rad/resources/schemas/guidestar-1.0.0.yaml:214]: I believe there's already a 'dec_uncertainty' field in WFIExposure and GuideWindow from file target-1.0.0.yaml

File: src/rad/resources/schemas/guidestar-1.0.0.yaml [hunk 5]
```diff
@@ -302,51 +322,56 @@ properties:
...
     type: number
     sdf:
       special_processing: VALUE_REQUIRED
       source:
         origin: TBD
     archive_catalog:
       datatype: float
-      destination: [WFIExposure.gs_mura, GuideWindow.gs_mura]
-  gs_mudec:
+      destination: [WFIExposure.proper_motion_ra, GuideWindow.proper_motion_ra]
```
Comment (kdupriestsci) [inline src/rad/resources/schemas/guidestar-1.0.0.yaml:339]: I believe there's already a 'proper_motion_ra' field in WFIExposure and GuideWindow from file target-1.0.0.yaml

File: src/rad/resources/schemas/guidestar-1.0.0.yaml [hunk 6]
```diff
@@ -302,51 +322,56 @@ properties:
...
     sdf:
       special_processing: VALUE_REQUIRED
       source:
         origin: TBD
     archive_catalog:
       datatype: float
-      destination: [WFIExposure.gs_mudec, GuideWindow.gs_mudec]
-  gs_para:
-    title: Guide Star Annual Parallax
+      destination: [WFIExposure.proper_motion_dec, GuideWindow.proper_motion_dec]
```
Comment (kdupriestsci) [inline src/rad/resources/schemas/guidestar-1.0.0.yaml:353]: I believe there's already a 'proper_motion_dec' field in WFIExposure and GuideWindow from file target-1.0.0.yaml

**Review by kdupriestsci — CHANGES_REQUESTED** [review]

**Review by schlafly — APPROVED** [review]
This all looks fine to me, though of course I defer to Kim on the database side!  Note that this also does unit-dropping, which is in Mairan's PR as well, and the plan is to include some unit information in the schema but not to enforce it with the quantity tag.  No action needed here, just flagging that the unit-related changes will see churn & conflicts soon.

File: src/rad/resources/schemas/guidestar-1.0.0.yaml (hunk 1 above)
Comment (PaulHuwe) [reply src/rad/resources/schemas/guidestar-1.0.0.yaml:207]: The `target-1.0.0.yaml` schema is being removed with this suite of PRs, so there should be no conflict.

File: src/rad/resources/schemas/guidestar-1.0.0.yaml (hunk 3 above)
Comment (PaulHuwe) [reply src/rad/resources/schemas/guidestar-1.0.0.yaml:220]: The `target-1.0.0.yaml` schema is being removed with this suite of PRs, so there should be no conflict.

File: src/rad/resources/schemas/guidestar-1.0.0.yaml (hunk 4 above)
Comment (PaulHuwe) [reply src/rad/resources/schemas/guidestar-1.0.0.yaml:214]: The `target-1.0.0.yaml` schema is being removed with this suite of PRs, so there should be no conflict.

File: src/rad/resources/schemas/guidestar-1.0.0.yaml (hunk 5 above)
Comment (PaulHuwe) [reply src/rad/resources/schemas/guidestar-1.0.0.yaml:339]: The `target-1.0.0.yaml` schema is being removed with this suite of PRs, so there should be no conflict.

File: src/rad/resources/schemas/guidestar-1.0.0.yaml (hunk 6 above)
Comment (PaulHuwe) [reply src/rad/resources/schemas/guidestar-1.0.0.yaml:353]: The `target-1.0.0.yaml` schema is being removed with this suite of PRs, so there should be no conflict.

**Review by jbrookens — APPROVED** [review]

**Pull request merged**